liberty-ls/src/main/java/io/openliberty/tools/langserver/utils/ServerPropertyValues.java

```diff
@@ -72,20 +73,50 @@ public class ServerPropertyValues {
         );
     }};
 
-    public static boolean canValidateKeyValues(LibertyTextDocument tdi, String key) {
+    // TODO: it seems specifying Integers above MAX_VALUE breaks all diagnostics...
```

Comment:
Is this TODO still true? Or did you manage to add the various `catches` that were needed and handle the exceptions appropriately?

Reply:
This TODO has been resolved 👍🏼